src/pages/index.astro

```diff
@@ -2,16 +2,22 @@
 import { Body } from "../components/index/body.tsx";
 import fetch from 'node-fetch';
 import * as cheerio from 'cheerio';
+import NetLogoApi from "../utils/api.js";
+
+const api = new NetLogoApi();
+
+const allData = await api.getAllData();
+console.log(allData);
 
 let base = 'https://backend.netlogo.org';
```

Comment:
replace base with BASE_URL somewhere as a constant for the whole app as discussed above